<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariablesEditor</class>
 <widget class="QFrame" name="VariablesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Form Class</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="gbOthersVariables">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Variables</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tbOthersVariablesAdd">
        <property name="toolTip">
         <string>Add...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/add.png</normaloff>:/listeditor/icons/listeditor/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tbOthersVariablesEdit">
        <property name="toolTip">
         <string>Edit...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/edit.png</normaloff>:/listeditor/icons/listeditor/edit.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="tbOthersVariablesRemove">
        <property name="toolTip">
         <string>Remove...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/remove.png</normaloff>:/listeditor/icons/listeditor/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QListWidget" name="lwOthersVariables">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOthersValues">
     <property name="title">
      <string>Values</string>
     </property>
     <layout class="QGridLayout" name="_3">
      <item row="0" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tbOthersValuesAdd">
        <property name="toolTip">
         <string>Add...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/add.png</normaloff>:/listeditor/icons/listeditor/add.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tbOthersValuesEdit">
        <property name="toolTip">
         <string>Edit...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/edit.png</normaloff>:/listeditor/icons/listeditor/edit.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="tbOthersValuesRemove">
        <property name="toolTip">
         <string>Remove...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/remove.png</normaloff>:/listeditor/icons/listeditor/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="tbOthersValuesClear">
        <property name="toolTip">
         <string>Clear...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/listeditor/icons/listeditor/clear.png</normaloff>:/listeditor/icons/listeditor/clear.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QListWidget" name="lwOthersValues">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
